eq-author/src/apollo/resolvers/index.js

```diff
@@ -0,0 +1,125 @@
+import gql from "graphql-tag";
+
+export const typeDefs = gql`
+  extend type QuestionPage {
+    isNew: Boolean!
+  }
+
+  extend type Mutation {
```

Comment:
Do we need to extend the mutations as well? I would assume we could just extend the type

Reply:
Apparently not. I've removed them